flashinfer/attention.py

```
@@ -66,6 +74,43 @@ def __init__(
             device=torch.device("cpu"),
             pin_memory=True,
         )
+        self.device = torch.device(device)
+
+        self._use_cuda_graph = use_cuda_graph
+        if use_cuda_graph:
+            if not torch.is_tensor(paged_kv_indptr_buffer):
+                raise ValueError(
+                    "paged_kv_indptr_buffer should be a torch.Tensor in cudagraph mode"
+                )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)